tools/Makefile

```diff
@@ -27,6 +27,7 @@ tools-y += sstrip make-ext4fs e2fsprogs mtd-utils mkimage
 tools-y += firmware-utils patch-image quilt padjffs2
 tools-y += mm-macros missing-macros cmake scons bc findutils gengetopt patchelf
 tools-y += mtools dosfstools libressl
+tools-y += ninja meson
```

Comment:
This change means, that everyone would be forced to (re)build ninja/meson every time, for no apparent reason. It would make ninja/meson very exposed as well, which would likely mean increased maintenance effort.